### PR TITLE
fix(v2): Fix Hebrew translations

### PR DESCRIPTION
**Pull request by nirtamir2**
## Motivation

Following https://github.com/facebook/docusaurus/pull/4587#discussion_r634415102 comment - This PR removes the Hebrew string for the number 2, and use it directly.
Example for English speakers:
`Two posts => 2 posts`
because "post" in Hebrew does not have a special form for two (unlike other Hebrew words like "day" יום / יומיים / ימים).

Also - after running it locally I fix other inconsistent translations I found.

image

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Test it locally

## Related PRs

https://github.com/facebook/docusaurus/pull/4811

### CODE REVIEW

File: packages/docusaurus-theme-classic/codeTranslations/he.json [hunk 1]
```diff
@@ -24,13 +24,13 @@
   "theme.blog.paginator.newerEntries": "הכי חדש",
   "theme.blog.paginator.olderEntries": "ישן יותר",
   "theme.blog.post.paginator.navAriaLabel": "רשימת חדשות של הבלוג",
-  "theme.blog.post.paginator.newerPost": "הכי חדש",
```
Comment (nirtamir2) [inline packages/docusaurus-theme-classic/codeTranslations/he.json:27]: Fix from "The newest"

File: packages/docusaurus-theme-classic/codeTranslations/he.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "theme.AnnouncementBar.closeButtonAriaLabel": "לסגור",
   "theme.CodeBlock.copied": "הועתק",
-  "theme.CodeBlock.copy": "להעתיק",
-  "theme.CodeBlock.copyButtonAriaLabel": "להעתיק קוד",
+  "theme.CodeBlock.copy": "העתק",
+  "theme.CodeBlock.copyButtonAriaLabel": "העתק קוד",
```
Comment (nirtamir2) [inline packages/docusaurus-theme-classic/codeTranslations/he.json:5]: Change from "to copy" to "copy" (in a form like "do copy" command)

File: packages/docusaurus-theme-classic/codeTranslations/he.json [hunk 3]
```diff
@@ -24,13 +24,13 @@
...
+  "theme.blog.post.paginator.newerPost": "חדש יותר",
   "theme.blog.post.paginator.olderPost": "ישן יותר",
-  "theme.blog.post.plurals": "רשומה אחת|שתי רשומות|{count} רשומות|{count} רשומות",
+  "theme.blog.post.plurals": "רשומה אחת|{count} רשומות|{count} רשומות|{count} רשומות",
   "theme.blog.post.readMore": "קרא עוד...",
-  "theme.blog.post.readingTime.plurals": "דקת קריאה|שתי דקות קריאה|{readingTime} דקות קריאה|{readingTime} דקות קריאה",
+  "theme.blog.post.readingTime.plurals": "דקת קריאה|{readingTime} דקות קריאה|{readingTime} דקות קריאה|{readingTime} דקות קריאה",
   "theme.blog.tagTitle": "{nPosts} עם התג \"{tagName}\"",
-  "theme.common.editThisPage": "לערוך דף",
+  "theme.common.editThisPage": "ערוך דף זה",
```
Comment (nirtamir2) [inline packages/docusaurus-theme-classic/codeTranslations/he.json:33]: "to edit page" => "edit this page" (in a form of command)

File: packages/docusaurus-theme-classic/codeTranslations/he.json [hunk 4]
```diff
@@ -40,15 +40,15 @@
   "theme.docs.sidebar.collapseButtonTitle": "סגור",
   "theme.docs.sidebar.expandButtonAriaLabel": "פתח",
   "theme.docs.sidebar.expandButtonTitle": "פתח",
-  "theme.docs.sidebar.responsiveCloseButtonLabel": "לסגור תפריט",
-  "theme.docs.sidebar.responsiveOpenButtonLabel": "לפתוח תפריט",
+  "theme.docs.sidebar.responsiveCloseButtonLabel": "סגור תפריט",
+  "theme.docs.sidebar.responsiveOpenButtonLabel": "פתח תפריט",
```
Comment (nirtamir2) [inline packages/docusaurus-theme-classic/codeTranslations/he.json:44]: Change from the action to open/top close to the command open / close

File: packages/docusaurus-theme-classic/codeTranslations/he.json [hunk 5]
```diff
@@ -40,15 +40,15 @@
...
   "theme.docs.versions.latestVersionLinkLabel": "גרסא אחרונה",
   "theme.docs.versions.latestVersionSuggestionLabel": "לדוקומנטאציה עדכנית, ראה {latestVersionLink} ({versionLabel}).",
   "theme.docs.versions.unmaintainedVersionLabel": "דוקומנטאציה זו {siteTitle} {versionLabel}, כבר לא נתמכת.",
   "theme.docs.versions.unreleasedVersionLabel": "דוקומנטאציה שעדיין לא יצאה לאוויר ל {siteTitle} {versionLabel} גרסא.",
   "theme.lastUpdated.atDate": " בתאריך {date}",
   "theme.lastUpdated.byUser": " על ידי {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "שונה{atDate}{byUser}",
+  "theme.lastUpdated.lastUpdatedAtBy": "עודכן{atDate}{byUser}",
```
Comment (nirtamir2) [inline packages/docusaurus-theme-classic/codeTranslations/he.json:51]: Use more appropriate word: 
Change => Update

**Review by nirtamir2 — COMMENTED** [review]
I added explainations for changes

**Comment by slorber** [comment]
Thanks !

It's not like I'm able to review the changes, so let's merge 😅

**Pull request merged**